dashboard/app/models/parental_permission_request.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: parental_permission_requests
+#
+#  id             :bigint           not null, primary key
+#  user_id        :integer          not null
+#  parent_email   :string(255)      not null
+#  uuid           :string(36)       not null
+#  reminders_sent :integer          default(0), not null
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#
+# Indexes
+#
+#  index_parental_permission_requests_on_user_id  (user_id)
+#  index_parental_permission_requests_on_uuid     (uuid)
+#
+class ParentalPermissionRequest < ApplicationRecord
+  before_create :set_uuid
+
+  def set_uuid
```

Comment:
Dangit, I wish I had done something similar in the lti_integration table. This is cool.

Reply:
There's still time! You can add it if you want ;)